### DISCUSSION TITLE
suggestion: copy image button

### DISCUSSION THREAD

**Discussion by 13c37** [flexbooru/flexbooru#195] [Ideas]
Heya,

I think a copy image button when you hold down on the image would be cool!

![poor example](https://user-images.githubusercontent.com/69517127/185254760-0f2f508c-4dc5-49fe-bf38-cc5862dccdc3.jpg)

**Comment by onlymash** [comment]
In info page, Long click the link item

c3 ***@***.***> 于 2022年8月18日周四 06:30写道：

**Reply by 13c37** [reply]
oh, thanks!
